src/cart/crt_iv.c

```diff
@@ -2062,8 +2227,8 @@ handle_ivsync_response(const struct crt_cb_info *cb_info)
 	struct iv_sync_cb_info	*iv_sync = cb_info->cci_arg;
 	struct crt_iv_ops	*iv_ops;
 
-	if (iv_sync->isc_bulk_hdl != CRT_BULK_NULL)
-		crt_bulk_free(iv_sync->isc_bulk_hdl);
+	DBG_ENTRY();
+	crt_bulk_free(iv_sync->isc_bulk_hdl);
```

Comment:
Same here, why is null bulk handle check removed?

Reply:
fixed